ruby/test/minitest/queue/order_reporter_test.rb

```diff
@@ -7,16 +7,41 @@ class OrderReporterTest < Minitest::Test
 
     def setup
       @reporter = OrderReporter.new(path: log_path)
+    end
+
+    def test_start
       @reporter.start
+      @reporter.report
+      assert_equal [], File.readlines(log_path).map(&:chomp)
     end
 
     def test_before_test
+      @reporter.start
       @reporter.before_test(runnable('a'))
       @reporter.before_test(runnable('b'))
       @reporter.report
       assert_equal ['Minitest::Test#a', 'Minitest::Test#b'], File.readlines(log_path).map(&:chomp)
     end
 
+    unless truffleruby?
+      def test_forking
```

Comment:
Do you mean to run this test only when not running the test suite with Truffle Ruby?
Does that mean the feature implemented in this PR is not compatible with Truffle Ruby?
Do we need a fallback? Will ci-queue not work with Truffle Ruby anymore?